### ISSUE TITLE
DLSSTweaks unable to force preset above 0.69 scale since DLSS 3.1.13

### ISSUE THREAD

**Issue by GitDoric** [emoose/DLSSTweaks#86]
Hi there :) !

Nvidia added DLAA as part of a DLSS quality preset in 3.1.13.
Since then, DLSSTweaks has been unable to force a preset to use when using 0.70+ scale and they now all default to preset F.
I'm using 0.75 & 0.85 scale and forcing presets on 3.1.11 and below work fine, 3.1.13 to 3.1.30 not anymore, so i'm almost sure this is linked to that.

I hope this will be looked upon and fixed in a new version, DLSSTweaks has been part of my tweak routine now and i can't stop using it :) !

Kind Regards !

**Comment by emoose** [comment]
For 3.1.11 and older it uses a hook to let us override the NV-provided default presets, looks like 3.1.13 changed the code around that stuff again though, so our hooks won't work properly now.

Looking into updating them, might have something to post soon, for now have you tried the OverrideAppId option at bottom of the INI? That was the old method to get around the NV presets before these hooks were added, hopefully might still work on these new versions.

**Comment by GitDoric** [comment]
> For 3.1.11 and older DLSSTweaks uses a hook to let us override the NV-provided default presets, looks like 3.1.13 changed the code around that stuff again though, so our hooks won't work properly now.
> 
> Looking into updating them, might have something to post soon, for now have you tried the OverrideAppId option at bottom of the INI? That was the old method to get around the NV presets before these hooks were added, hopefully might still work on these new versions.
> 
> Alternately if that doesn't work you might be able to try overriding preset via NvProfileInspector with the XML from [Orbmu2k/nvidiaProfileInspector#156 (comment)](https://github.com/Orbmu2k/nvidiaProfileInspector/issues/156#issuecomment-1661197267), unfortunately that would apply it to all DLSS 3.1+ games on your system though.

Yes, I should have mentioned that i had to use OverrideAppId to be able to force a preset with 3.1.13+ ;  It's not working at all if i don't use that option.
However OverrideAppId is not able to do anything for the 0.70+ scale, seems as you say, the code has been changed too much because of the new DLAA preset added since 3.1.13.
I think that 0.70+ is now hooked to the new dlaa preset or may be a new "high quality" one ? 
Pre 3.1.13 all 0.66+ scale up to 1.0 was considered "quality" mode.

**Comment by achtchaern** [comment]
@GitDoric What exactly did you have to configure regarding OverrideAppId? I can't find instructions on this setting. I tried "OverrideAppId=0xe658703", which should be a generic app, but that didn't work: In Portal RTX, the AppId stayed the same. My goal is to change the DLSS profile game-specifically (so the global option via Nvidia Profile Inspector is not an option for me).

**Comment by achtchaern** [comment]
Thank you for this discussion! It would be great if we could set the preset with scaling > 0.70 again.

**Comment by GitDoric** [comment]
emoose uploaded a new beta version : https://github.com/emoose/DLSSTweaks/files/12333444/DLSSTweaks_0.200.7-beta1.zip
After a quick try it seems to have fixed the issue !
I can now force my preset D at 0.85x scale, YAY !

Thanks a lot to you dev 🥰.

**Comment by achtchaern** [comment]
Thank you! Looking forward to trying this out.

**Comment by achtchaern** [comment]
Unfortunately, it does not work for me in Portal with RTX. Tested with 0.200.7 beta 1 and 2a. It still forces preset F above 0.70
